import CustomError from "./CustomError";

export default class ClienteNaoEncontrado extends CustomError {
  constructor() {
    super("Cliente n√£o encontrado", 404);
  }
}
